website/docs/Reference/telemetry.md

```diff
@@ -80,6 +80,18 @@ Telemetry can be disabled using any of the following options:
    until the `donottrack` string is removed.
    :::
 
+5. Use the `db.disableFreeMonitoring()` command on runtime.
+
+   ```sh
+   test> db.disableFreeMonitoring()
+   { ok: 1 }
+   ```
```

Comment:
I look at the other examples and it seems that for consistency it should be something like
```suggestion
   ```js
db.disableFreeMonitoring()
   ```
```

(with no response example)